apps/src/templates/manageStudents/ManageStudentsAgeCell.jsx

```diff
@@ -11,10 +11,16 @@ class ManageStudentAgeCell extends Component {
     editedValue: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
     // Provided by redux
     editStudent: PropTypes.func.isRequired,
+    setSharingDefault: PropTypes.func.isRequired,
   };
 
+  // For privacy reasons, we disable sharing by default if the student is
+  // under the age of 13 if the age was previously not set.
   onChangeAge = (e) => {
     this.props.editStudent(this.props.id, {age: e.target.value});
+    if (this.props.age === '') {
```

Comment:
I think you'll need to put this behind a flag until we launch. We don't want users accidentally updating the share setting yet.

Reply:
Good point - did not think of that! I'll put behind the flag.